base/linalg/schur.jl

```diff
@@ -107,6 +107,7 @@ function schur(A::StridedMatrix)
     SchurF = schurfact(A)
     SchurF[:T], SchurF[:Z], SchurF[:values]
 end
+schur(A::AbstractMatrix) = schur(full(A))
```

Comment:
Isn't `full(A)` always of type `StridedMatrix`? If not, I'll lot into it because just adding a  command like that (`Symmetric, Hermitian, Triangular` etc) in shur.jl, produces UndefVarError.